app/models/site.rb

```diff
@@ -27,6 +27,10 @@ def credit_enabled?
     participants.exists? && credit_roles.exists?
   end
 
+  def participant_enabled?
```

Comment:
Hmm, `participant_enabled?` isn't  good name I think.
Because they look like for whole participants features. (editor/participants)

How about `participant_description_page_enabled?`